Source/relay/TourGuide/Paths/Avatar of West of Loathing.ash

```diff
@@ -143,7 +143,7 @@ void PathAvatarOfWestOfLoathingGenerateTasks(ChecklistEntry [int] task_entries,
     class_points[$class[Snake Oiler]] += get_property_int("awolPointsSnakeoiler");
     
     item [class] tale_for_class;
-    tale_for_class[$class[Cow Puncher]] = $item[Tales of the West: Cow Punching];
+    tale_for_class[$class[Cow Puncher]] = $item[Tales of the West: \ Cow Punching];
```

Comment:
wait there's an extra space in the item name for this? that's funny and stupid lol

Reply:
Yes it's very weird. I've also noticed that the warning shows up in other scripts now too, autoscend for example. Feels like the item name changed internal to mafia or something? Not clear. But this does remove the warning.